static/js/reducers/financial_aid_test.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import configureTestStore from 'redux-asserts';
+import { assert } from 'chai';
+import sinon from 'sinon';
+
+import {
+  START_CALCULATOR_EDIT,
+  startCalculatorEdit,
+  CLEAR_CALCULATOR_EDIT,
+  clearCalculatorEdit,
+  UPDATE_CALCULATOR_EDIT,
+  updateCalculatorEdit,
+  UPDATE_CALCULATOR_VALIDATION,
+  updateCalculatorValidation,
+  REQUEST_ADD_FINANCIAL_AID,
+  requestAddFinancialAid,
+  RECEIVE_ADD_FINANCIAL_AID_SUCCESS,
+  RECEIVE_ADD_FINANCIAL_AID_FAILURE,
+  addFinancialAid,
+} from '../actions/financial_aid';
+import {
+  FETCH_FAILURE,
+  FETCH_PROCESSING,
+  FETCH_SUCCESS,
+} from '../actions';
+import { setCurrentProgramEnrollment } from '../actions/enrollments';
+import { FINANCIAL_AID_EDIT } from './financial_aid';
+import rootReducer from '../reducers';
+import * as api from '../util/api';
+
+describe('financial aid reducers', () => {
+  let sandbox, store, dispatchThen;
+  let addFinancialAidStub;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    store = configureTestStore(rootReducer);
+    dispatchThen = store.createDispatchThen(state => state.financialAid);
+    store.dispatch(setCurrentProgramEnrollment(1));
+    addFinancialAidStub = sandbox.stub(api, 'addFinancialAid');
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('should let you start editing', () => {
+    return dispatchThen(startCalculatorEdit(1), [START_CALCULATOR_EDIT]).then(state => {
+      let expectation = Object.assign({}, FINANCIAL_AID_EDIT, {
+        programId: 1
+      });
+      assert.deepEqual(state, expectation);
+    });
+  });
+
+
+  it('should let you clear edits', () => {
+    store.dispatch(startCalculatorEdit(1));
+    return dispatchThen(clearCalculatorEdit(), [CLEAR_CALCULATOR_EDIT]).then(state => {
+      assert.deepEqual(state, {});
```

Comment:
Can you change to `assert.deepEqual(state, INITIAL_FINANCIAL_AID_STATE)` to be explicit?